api/client/stats.go

```diff
@@ -71,8 +71,12 @@ func (s *containerStats) Collect(cli *DockerCli, streamStats bool) {
 			s.Memory = float64(v.MemoryStats.Usage)
 			s.MemoryLimit = float64(v.MemoryStats.Limit)
 			s.MemoryPercentage = memPercent
-			s.NetworkRx = float64(v.Network.RxBytes)
-			s.NetworkTx = float64(v.Network.TxBytes)
+			// Note: The cgroup network Rx and Tx are flipped compared to ifconfig
```

Comment:
I am running Docker 1.7.1 and I see that the Rx/Tx is in wrong order. If you are in doubt of this change and maybe plan to fix it on a lower level (stats API) I can clean out this change from the PR so that you can accept the rest of the PR.